### PR TITLE
Add flask: one of the most popular web framework for python

### PR DESCRIPTION
**Pull request by xpzouying**
Flask: The Python micro framework for building web applications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Zulip](https://github.com/zulip/zulip/labels/good%20first%20issue) _(label: good first issue)_ <br> Powerful open source group chat.
 - [cookiecutter](https://github.com/audreyr/cookiecutter/labels/low-hanging-fruit) _(label: low-hanging-fruit)_ <br> A command-line utility that creates projects from cookiecutters (project templates). E.g. Python package projects, jQuery plugin projects.
 - [django cookiecutter](https://github.com/pydanny/cookiecutter-django/labels/hacktoberfest) _(label: django hacktober special)_ <br> An implementation of Python for backend web development.
+- [Flask](https://github.com/pallets/flask/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> The Python micro framework for building web applications.
```
Comment (vvscode) [inline README.md]: check format for links on surronding examples

Comment (xpzouying) [reply README.md]: Hi @vvscode ,

I don't understand. I already followed the rule of format in `CONTRIBUTING.md`.

Comment (MunGell) [reply README.md]: Not exactly, the link should look like this:

```suggestion
- [Flask](https://github.com/pallets/flask/labels/good%20first%20issue) _(label: good first issue)_ <br> The Python micro framework for building web applications.
```

File: README.md [hunk 2]
```diff
@@ -182,9 +182,11 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Zulip](https://github.com/zulip/zulip/labels/good%20first%20issue) _(label: good first issue)_ <br> Powerful open source group chat.
 - [cookiecutter](https://github.com/audreyr/cookiecutter/labels/low-hanging-fruit) _(label: low-hanging-fruit)_ <br> A command-line utility that creates projects from cookiecutters (project templates). E.g. Python package projects, jQuery plugin projects.
 - [django cookiecutter](https://github.com/pydanny/cookiecutter-django/labels/hacktoberfest) _(label: django hacktober special)_ <br> An implementation of Python for backend web development.
+- [Flask](https://github.com/pallets/flask/labels/good%20first%20issue) _(label: good first issue)_ <br> The Python micro framework for building web applications.
 - [wemake-python-stylegide](https://github.com/wemake-services/wemake-python-styleguide/labels/level%3Astarter) _(label: level:starter)_ <br> The strictest and most opinionated python linter ever!
 - [Ansible](https://github.com/ansible/ansible/labels/easyfix) _(label:easyfix)_ <br> A simple IT automation platform
 
+
```
Comment (vvscode) [inline README.md]: do we really need this?

**Pull request closed without merging**